reference/troubleshooting/troubleshoot-multi-stage-query-engine.md

```diff
@@ -200,6 +201,14 @@ See [null handling support](null-value-support.md)
 
 * aggregate function that requires literal input (such as `percentile`, `firstWithTime`) might result in a non-compilable query plan when used in v2.&#x20;
 
+### Byte literals
+
+<!-- TODO: Explain how to use byte literals -->
```

Comment:
Are you planning to fill this part in this PR or a separate PR?

Reply:
Yes, this two TODOs shouldn't be here. I'm going to remove them

Reply:
I've decided to just add a short explanation for these two topics